src/cli/commands/db.py

```
@@ -0,0 +1,188 @@
+        print_db_status(status)
+
+
+@app.command("clear")
+def clear_database(
+    force: bool = typer.Option(
+        False,
+        "--force",
+        "-f",
+        help="Skip confirmation prompt",
+    ),
+    keep_schema: bool = typer.Option(
```

Ruff (0.14.10)

Unused function argument: `keep_schema`

(ARG001)